proxy/src/control/destination/background.rs

```diff
@@ -60,7 +60,7 @@ struct DestinationSet<T: HttpService<ResponseBody = RecvBody>> {
     addrs: Exists<Cache<SocketAddr, Metadata>>,
     query: Option<DestinationServiceQuery<T>>,
     dns_query: Option<IpAddrListFuture>,
-    txs: Vec<mpsc::UnboundedSender<Update>>,
+    responds: Vec<Respond>,
```

Comment:
Something about this naming... I feel a mental stumble when reading it :D

Reply:
`responders`?